content/docs/reference/types/object.mdx

```diff
@@ -1,12 +1,16 @@
 ---
 title: The "object" field
-last_edited: '2024-09-12T01:50:34.111Z'
+last_edited: '2024-09-26T06:54:19.563Z'
 next: content/docs/reference/types/rich-text.mdx
 previous: content/docs/reference/types/reference.mdx
 ---
 
 ## `object` Type
 
+The `object` field is a dynamic field type creates a nested page in the CMS with the data it contains.
```

Comment:
"a dynamic field type **that** creates a nested page in the CMS with the data it contains"

Reply:
Good catch